src/auth/withPermissionsFilteredChildren.js

```diff
@@ -42,21 +42,19 @@ export default BaseComponent => {
             const { authClient, ...props } = this.props;
             return (
                 <BaseComponent {...props}>
-                    {children && Array.isArray(children) ? (
```

Comment:
please revert this change, as well as all the changes due to prettier configuration change